packages/server/src/api/controllers/table/internal.ts

```diff
@@ -18,10 +18,11 @@ export async function save(ctx: UserCtx<SaveTableRequest, SaveTableResponse>) {
     _rename?: RenameColumn
   } = {
     _id: generateTableID(),
-    ...rest,
-    type: "table",
-    sourceType: TableSourceType.INTERNAL,
     views: {},
```

Comment:
it would probably be better to move this outside of the block, as if it is moved after `...rest` then it'll get overwritten, we should probably do something like 
```
if (!tableToSave.views) {
  tableToSave.views = {}
}
```

Reply:
If this was moved again, the tests should catch it. But I can move it out, to make it more explicit